File: Fast-&-Lethal/tigershark/TigerShark.java
// TigerShark.java
public class TigerShark {
    private String name = "TigerShark";
    private String role = "Slayer";
    private String lane = "Oblivion";
    private String resource = "Essência do Void";

    public void passive() {
        System.out.println("Sede de Sangue: aumenta o dano conforme o inimigo perde vida.");
    }

    public void abilityQ() {
        System.out.println("Corte Predatório: investida que causa dano verdadeiro e sangramento.");
    }

    public void abilityW() {
        System.out.println("Cauda Cortante: giro com a cauda que afasta os inimigos e aplica lentidão.");
    }

    public void abilityE() {
        System.out.println("Rugido das Profundezas: silencia os inimigos e revela invisíveis.");
    }

    public void ultimate() {
        System.out.println("Mergulho Mortal: salta e atinge todos os inimigos na área com dano massivo.");
    }
}

File: Fast-&-Lethal/tigershark/tiger_shark.py
# tiger_shark.py

class TigerShark:
    def __init__(self):
        self.name = "TigerShark"
        self.role = "Slayer"
        self.lane = "Oblivion"
        self.resource = "Essência do Void"

    def passive(self):
        return "Sede de Sangue: aumenta o dano conforme o inimigo perde vida."

    def ability_q(self):
        return "Corte Predatório: investida que causa dano verdadeiro e sangramento."

    def ability_w(self):
        return "Cauda Cortante: giro com a cauda que afasta os inimigos e aplica lentidão."

    def ability_e(self):
        return "Rugido das Profundezas: silencia os inimigos e revela invisíveis."

    def ultimate(self):
        return "Mergulho Mortal: salta e atinge todos os inimigos na área com dano massivo."

# Exemplo de uso
if __name__ == "__main__":
    ts = TigerShark()
    print(ts.passive())
    print(ts.ability_q())
    print(ts.ability_w())
    print(ts.ability_e())
    print(ts.ultimate())